core/translate/expr.rs

```diff
@@ -1552,6 +1552,112 @@ pub fn translate_expr(
                         }
                     }
                 }
+                Func::Math(math_func) => match math_func {
+                    // Unary math functions
+                    MathFunc::Acos
```

Comment:
Maybe doing an

```
impl MathFunc {
  arity(&self) -> Arity::Nullary | Arity::Unary | Arity::Binary
}
```

might be worth it since this pattern matching is done in two places and theres no special handling of specific functions